metrics/tests/site-startup.test.ts

```diff
@@ -30,18 +29,6 @@ test.describe( 'Startup Metrics', () => {
 		} );
 
 		await session.cleanup();
-
-		const video = session.mainWindow.video();
```

Comment:
I actually wonder if we should keep this somehow. But I agree it's better to remove the current tmp code :)

Reply:
Anyhow, let's implement in a separate PR!